gradle/settings/parameters.gradle

```diff
@@ -10,4 +10,5 @@ gradle.ext.remoteApiProjectPath=":remoteapi:labkey-api-java"
 gradle.ext.jdbcApiProjectPath=":remoteapi:labkey-api-jdbc"
 gradle.ext.schemasProjectPath=":server:modules:platform:api"
 gradle.ext.testProjectPath=":server:testAutomation"
+gradle.ext.minificationProjectPath=":server:minification"
```

Comment:
This isn't really necessary since this is the default value anyway and I don't expect anyone is going to change this.

Reply:
I was just adding it to be consistent with the other `*ProjectPath` parameters.